data/darktableconfig.xml.in

```diff
@@ -2613,6 +2613,56 @@
     <shortdescription/>
     <longdescription/>
   </dtconfig>
+  <dtconfig>
+    <name>plugins/imageio/format/jxl/bpp</name>
+    <type>
+      <enum>
+        <option>8</option>
+        <option>10</option>
+        <option>12</option>
+        <option>16</option>
+        <option>32</option>
+      </enum>
+    </type>
+    <default>8</default>
```

Comment:
I know nothing about JXL... But won't 16bits or above be a nice default to have better quality by default and move away from standard jpeg 8bits?

Reply:
We don't do it for AVIF, TIFF, JPEG 2000, PNG either, and they can all do better than 8b - we always start w/ the lowest common denominator and let the user make a conscious decision. I see no real need to break w/ that logic?

Besides, the JPEG XL quality should still be better, even for 8b reconstruction (internal storage implementation is different).

Reply:
Ok by me, just making sure all options are discussed.